### PR TITLE
Hook SDL scripts to run part of Post-Build Validate step in Arcade

### PR DESCRIPTION
**Pull request by sunandabalu**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -188,3 +188,16 @@ stages:
       # Sourcelink validation isn't passing for Arcade due to some regressions. This should be
       # enabled back once this issue is resolved: https://github.com/dotnet/arcade/issues/2912
       enableSourceLinkValidation: false
+      # This is to enable SDL runs part of Post-Build Stage
```
Comment (JohnTortugo) [inline azure-pipelines.yml]: "run SDK as part of the Post-Build **Validation** stage"

File: azure-pipelines.yml [hunk 2]
```diff
@@ -188,3 +188,16 @@ stages:
...
+      SDL:
```
Comment (JohnTortugo) [inline azure-pipelines.yml]: I think it would better if we renamed this to "SDLValidationParameters"

**Comment by JohnTortugo** [comment]
Since you're already enabling this to Arcade did you run a test build for it?

**Comment by sunandabalu** [comment]
> Since you're already enabling this to Arcade did you run a test build for it?

I validated this in arcade-validation - https://dev.azure.com/dnceng/internal/_build/results?buildId=232334&view=results

**Pull request merged**